integration/kube_integration_test.go

```diff
@@ -89,12 +90,10 @@ func newKubeSuite(t *testing.T) *KubeSuite {
 	if ok, _ := strconv.ParseBool(testEnabled); !ok {
 		t.Skip("Skipping Kubernetes test suite.")
 	}
-
 	suite := &KubeSuite{
-		kubeConfigPath: os.Getenv(teleport.EnvKubeConfig),
+		kubeConfigPath: os.Getenv("KUBECONFIG"),
```

Comment:
This change seems unnecessary - `teleport.EnvKubeConfig` is already defined as `"KUBECONFIG"` - perhaps a leftover from changing back from the hard-coded filename?